src/ranged.h

```diff
@@ -65,6 +65,19 @@ class target_handler
                                          const target_callback &on_ammo_change = target_callback() );
 };
 
+struct ranged_skill_data {
+    ranged_skill_data()
+    {}
```

Comment:
Why not `= default;` instead of `{}`? In fact, why specify the constructor at all? The compiler generated one is just as good.